app/server/utils/user.py

```diff
@@ -389,7 +390,20 @@ def proccess_create_or_modify_user_request(
     transfer_account_name = attribute_dict.get('transfer_account_name')
     first_name = attribute_dict.get('first_name')
     last_name = attribute_dict.get('last_name')
-    business_usage_id = attribute_dict.get('business_usage_id')
+
```

Comment:
Unless i'm misunderstanding how this works, the way in which an "other" usage gets added feels a bit odd. Like rather than having this deep concept of "other" that threads through the backend that requires the creation of an "other" category that isn't actually ever used, wouldn't it make more sense for the api have only one business usage field, and then if you provide a string that isn't an existing usage it creates a new one. Then on the front end, the concept of other would be constrained just to the UI of the drop down.

Reply:
yeah agreed, we should not have an other category anymore in itself. changed behavior to reflect this, but haven't tested it since my dev environment is broken

Reply:
k was able to test that it works now. change is ready for merging